cmd/list_ttps_test.go

```diff
@@ -0,0 +1,54 @@
+/*
+Copyright © 2023-present, Meta Platforms, Inc. and affiliates
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
+*/
+
+package cmd_test
+
+import (
+	"path/filepath"
+	"testing"
+
+	"github.com/facebookincubator/ttpforge/cmd"
+	"github.com/stretchr/testify/require"
+)
+
+func TestListRepos(t *testing.T) {
+	testConfigFilePath := filepath.Join("test-resources", "test-config.yaml")
+	testCases := []struct {
+		name      string
+		ttpRef    string
+		wantError bool
+	}{
+		{
+			name: "no-filter",
+		},
+	}
+
+	for _, tc := range testCases {
+		t.Run(tc.name, func(t *testing.T) {
+			rc := cmd.BuildRootCommand()
+			rc.SetArgs([]string{"list", "ttps", "-c", testConfigFilePath})
```

Comment:
I need to do some surgery on the stdout capture handling for testing (tricky to do it in a truly thread safe manner, cobra has an idiomatic way to do it that i'll implement) - will add assertions against stdout in separate PR

Reply:
done https://github.com/facebookincubator/TTPForge/issues/315